ext/AvizExt/viz/data_envelopment.jl

```diff
@@ -0,0 +1,133 @@
+using ADRIA.analysis: DEAResult
+
+"""
+    ADRIA.viz.data_envelopment_analysis(rs::ResultSet, DEA_output::DEAResult;axis_opts=Dict(),
```

Comment:
This docstring doesn't match the function signature (all `opts` params types are wrong) **and** it should be one method per line (this is how we do in the resto of this project at least).